piv/piv.go

```diff
@@ -140,16 +157,25 @@ type client struct {
 }
 
 func (c *client) Cards() ([]string, error) {
```

Comment:
nit: instead of having "client" and "Client" just have "Client" and unexport the cards method

So

```
type Client struct {
    Shared bool
}
func (c *Cilent) cards() ([]string, error)
func (c *Cilent) Open(card string) (*YubiKey, error)
```